dapr/proto/common/v1/common.proto

```diff
@@ -122,4 +122,35 @@ message StateOptions {
 
   StateConcurrency concurrency = 1;
   StateConsistency consistency = 2;
-}
\ No newline at end of file
+}
+
+// Configuration represents all the configuration items of specified application.
+message Configuration {
+  // Required. The name of configuration item
+  string app_id = 1;
+
+  // Required. The configuration store name to get and save this configuration document.
+  string store_name = 2;
+
+  // Required. The revision of this configuration.
+  // It should be an incremental integers and changed when configuration is updated.
+  string revision = 3;
+
+  // Configuration Item list.
+  repeated ConfigurationItem items = 4;
```

Comment:
Instead of many configuration items, can this be just a single string content? I remember a discussion where we could have the whole config to be JSON and it would enable multiple levels natively. The SDKs and runtime would need to validate the JSON content but it seems to be more extensible than providing just one level.